workers/unity/Assets/Fps/Scripts/Editor/SnapshotGenerator/SnapshotMenu.cs

```diff
@@ -25,6 +25,7 @@ private static void GenerateFpsSnapshot()
         private static Snapshot GenerateDefaultSnapshot()
         {
             var snapshot = new Snapshot();
+            snapshot.AddEntity(new EntityId(1), FpsEntityTemplates.CreateLoadBalancingPartition());
```

Comment:
Is it required to be 1, or does any ID work?

Reply:
It can be any entity ID, but this is referenced in the `BaseTemplate()` which is used for all entities. 

This ensures that the load balancing worker can claim this partition and then get authority over `AuthorityDelegation` for all entities.

Reply:
It's not very clear that it NEEDS to be 1, and is linked with the number in `BaseTemplate`

Reply:
Made this more explicit in https://github.com/spatialos/gdk-for-unity-fps-starter-project/pull/298/commits/05e3fe19d3e362a68674aa82f485c9eb7c03d672